### ISSUE TITLE
Docker v1.12 does not start

### ISSUE THREAD

**Issue by mkrech10** [chrisgahlert/gradle-dcompose-plugin#3]
I'm trying to use your gradle plug-in and I have docker version 1.12.  I get an error, "starting container with HostConfig was deprecated since v1.10 and removed in v1.12".  I'm wondering if you are aware of this issue and if you have any plans on fixing it.

**Comment by chrisgahlert** [comment]
Hi, thank you for the info. I'll look right into that!

**Comment by chrisgahlert** [comment]
I'm afraid it is a problem with the underlying _docker-java_ library: 
https://github.com/docker-java/docker-java/issues/617

We'll have to wait for them to fix this issue in order to support Docker 1.12.0.

**Comment by lordofthejars** [comment]
You can set the version field to 1.23 and it works.

**Comment by mkrech10** [comment]
lordofthejars - Can you explain how to set the version field please?

**Comment by lordofthejars** [comment]
In the plugin I don t know but in docker java client builder there is a
withApiVersion that allows to do that

**Comment by chrisgahlert** [comment]
In the plugin you can use the following:

``` gradle
dcompose {
  dockerClientConfig = {
    withApiVersion '1.23'
  }
}
```

But I'm in the process of releasing a new plugin version that always uses the lowest possible API version. (The compatibility tests are running right now!)

**Comment by mkrech10** [comment]
THAT WORKED!!!  Thanks guys!

**Comment by lordofthejars** [comment]
You are welcome, if you use docker 1.12 you need to use 1.23 or doesn t
work. This is the rule :)

**Comment by chrisgahlert** [comment]
I just released the new version **0.4.0** of the plugin. In this version the Docker API version field is explicitly set and it is not necessary anymore to use the fix from above.

Thank you @mkrech10 and @lordofthejars for helping fixing this!
